test/llvm/StdInOut.js

```diff
@@ -0,0 +1,44 @@
+// stdout: hello world? no
+
+(function() {
+  let getchar = __abstract(":integral", "getchar");
+  let putchar = __abstract(":integral", "putchar");
```

Comment:
Here's how you use it to call standard library functions. Didn't need any changes to the API.